### PR TITLE
Feat/review/user

### PR DESCRIPTION
**Pull request by xsijin**

### CODE REVIEW

File: src/components/BreadCrumb/BreadCrumb.jsx [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (chunxtan) [inline src/components/BreadCrumb/BreadCrumb.jsx:1]: can this file be deleted? i think it's also causing a merge conflict

**Review by chunxtan — CHANGES_REQUESTED** [review]
delete breadcrumb.jsx if it exists!

File: src/components/BreadCrumb/BreadCrumb.jsx (hunk 1 above)
Comment (xsijin) [reply src/components/BreadCrumb/BreadCrumb.jsx:1]: yes the breadcrumb shld've been deleted on an earlier PR (wasn't approved at that point) but in the later PR it wasn't (as git pull was still retrieving that file) hence the conflicts T-T

**Review by chunxtan — APPROVED** [review]
lgtm

**Pull request merged**